### PR TITLE
New flow documentation

### PR DESCRIPTION
**Pull request by BupycHuk**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,10 +50,13 @@ By default, the image name is
...
-Once merged, our CI will automatically publish these changes to the dev
-environment.
+They are independent of each other. Which means that to get your changes on both environments you have to create 2 different branches.
+Each of them should be based on the branch related to the environment you want to deploy and PRs should be created to the same branch. 
 
-To publish changes to the production environment, please ask responsible person
-to deploy your docker image manually.
\ No newline at end of file
+e.g.: To publish your changes to the `development` environment, you should create new branch from `main` branch and then create a PR to get your changes merged to the `main` branch.  
+Once your PR is merged, our CI will automatically publish these changes to corresponding environment.
```
Comment (nmarukovich) [inline README.md:62]: Does it work for production already?

Comment (BupycHuk) [reply README.md:62]: Not yet, but I made changes in advance.

**Review by nmarukovich — APPROVED** [review]

**Review by hors — APPROVED** [review]

**Review by tplavcic — CHANGES_REQUESTED** [review]
My only concern is if we have two completely separate branches main | production and start cherry-picking can it happen that these branches will somehow diverge over time if we are not careful?

**Comment by BupycHuk** [comment]
@tplavcic yes, that's possible, but we always can compare development and production branches to fix these problems.

**Pull request merged**